daml-script/export/src/test/scala/com/daml/script/export/AstSyntax.scala

```diff
@@ -14,9 +14,7 @@ private[export] object AstSyntax {
     }
   }
   def tuple(tys: Ast.Type*): Ast.Type = {
-    val tupleName = Ref.TypeConName.assertFromString(
-      "40f452260bef3f29dede136108fc08a88d5a5250310281067087da6f0baddff7:DA.Types:Tuple"
-    )
+    val tupleName = StablePackage.DA.Types.assertIdentifier("Tuple" + tys.length.toString)
```

Comment:
Tuples only go up to `Tuple20`. Maybe worth adding a comment somewhere and/or throwing an exception?

Reply:
The converters from daml-script/trigger are all quite "funky". 
We should probably go through all of them, not worthwhile to add a comment specifically here. 
I create #13694